src/status_im/translations/en.cljs

```diff
@@ -186,7 +190,9 @@
    :wallet-set-up-confirm-title          "Wrote it down?"
    :wallet-set-up-confirm-description    "You won’t be able to see your 3-word transaction phrase again after this."
    :wallet-backup-seed-title             "Backup your Seed Phrase"
+   :wallet-backup-recovery-title         "Backup your Recovery Phrase"
    :wallet-backup-seed-description       "This will help you to keep your money safe"
+   :wallet-backup-recovery-description   "This will help you to keep your money safe"
```

Comment:
why is it duplicated?

Reply:
Because there was a reference to `seed` in the key even if nothing was changed in the value.